my-app/src/Components/MovieList.tsx

```diff
@@ -0,0 +1,19 @@
+import { Link } from "react-router-dom";
+import IMovie from "../Types/Imovie";
+
+const MovieList = ({ movie }: { movie: IMovie }) => {
+  return (
+    <div className="card" style={{ width: "80%" }}>
+      <h3 className="card-title" key={movie.id}>
+        <Link to={`/moive/${movie.id}`}> {movie.title}</Link>
```

Comment:
change '/moive' to '/movie'

```suggestion
        <Link to={`/movie/${movie.id}`}> {movie.title}</Link>
```